src/helper/default.js

```diff
@@ -0,0 +1,3 @@
+/* eslint-disable import/prefer-default-export */
```

Comment:
I specifically said `helpers` not `helper`. Please update

Reply:
Okay i would